README.md

```diff
@@ -501,6 +501,7 @@ We also recommend these lists.
 * [Reply](https://metacpan.org/pod/Reply) - Read-eval-print-loop(REPL) command-line tool.
 * [Riji](https://metacpan.org/pod/Riji) - Static site generator using markdown and git mainly for blogging.
 * [Smart::Comments](https://metacpan.org/pod/Smart::Comments) - Comments that do more than just sit there.
+* [Explore Perl](https://kandi.openweaver.com/explore/perl) - Discover & find a curated list of popular & new libraries, top authors, trending project kits, discussions, tutorials & learning resources on kandi.
```

Comment:
I think it is "site", It is not is a module or tool created in Perl.

So I think not "tool section".

I could think of adding new H2 section to make it "other useful resource/information sites" or something like that.

Reply:
Agree @uzulla . Adding under a new section like 'Other Useful Resources / Information" makes sense. 
Let me know if you want us to do it that way with a new pull request?

Reply:
I'll be merge If you add commit and push this PR. 
Could you add section in bellow `# Other Awesome Lists` like as `# other useful resource/information sites`
thanks!

@kandikits